doc/chapters/gateways/10-environment-gateway.rst

```diff
@@ -74,4 +74,14 @@ to e.g. ``/tmp/test``, the varaiable value would now be set to ``/tmp/test:/some
 If ``SOME_ENVIRONMENT_VARIABLE`` had not been previously set, the value would now be
 set to ``:/some/path``.
 
+Important Note
+--------------
 
+Environment variables are internally stored as an array of pointers to strings in Linux based
+systems. This array is terminated by a NULL pointer. Each name and value pairs are matched with "="
```

Comment:
Yeah but when the system is reading this from environ, it will think as it is the end of environment table. I can refer http://linuxdocs.org/HOWTOs/Secure-Programs-HOWTO/environment-variables.html for more information. Please check part 4.2.2. paragraph 2